index.js

```diff
@@ -81,14 +81,14 @@ class RNCallKeep {
     }
   };
 
+  reportEndCallWithUUID = (uuid, reason) => {
+    RNCallKeepModule.reportEndCallWithUUID(uuid, reason);
+  }
+
   endCall = (uuid) => {
     RNCallKeepModule.endCall(uuid);
   };
 
-  rejectCall = (uuid) => {
```

Comment:
Do we really want to remove rejectCall as a function? It seems useful and could be shorthand for disconnect with reject cause + shut down the connection

Reply:
Seemed unnecessary with reportEndCallWithUUID but it can be added back